bootstrap.sh

```diff
@@ -67,8 +67,19 @@ sed_i() {
   fi
 }
 
-# update project name in database.yml
-sed_i "s/rails_api_base/${project_name}/g" config/database.yml
+# Define the file types to search for
+file_types=("yml" "yaml" "erb" "rb" "json")
+
+for file_type in "${file_types[@]}"
+do
+  # Find and replace the string in files of the current type, and save a backup file
+  find . -name "*.$file_type" -exec sed -i.bak "s|rails_api_base|${project_name}|g" {} +
```

Comment:
```suggestion
  find . -name "*.$file_type" -exec sed -i "" "s|rails_api_base|${project_name}|g" {} +
```

with this change the `.bak` files don't get created and we don't need to remove them below

Reply:
Actually, I think you need to use the `sed_i` function [defined above](https://github.com/rootstrap/rails_api_base/pull/591/files#diff-99d5a2a8ca4290b61836b3ad37941f2b925e9ff3151c90dcd5ea82364e4122a1L62)

Reply:
cc @guillermoap